### PR TITLE
added chipGroupMain div, updated snapshots

### LINKED ISSUE #4404: ChipGroups: close chip group button wraps at smaller widths
Any react work needed to after https://github.com/patternfly/patternfly/issues/3182 is completed.

Reproducible in the React Toolbar documentation for the Toolbar with filters example.

image

image

### PR DESCRIPTION
**Pull request by evwilkin**
**What**: Closes #4404 

Adds new `<div class="pf-c-chip-group__main">` wrapper element which keeps chip group close button in top-right corner instead of wrapping to new line.

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -209,7 +209,7 @@
     "@babel/helper-explode-assignable-expression" "^7.10.4"
     "@babel/types" "^7.10.4"
 
-"@babel/helper-builder-react-jsx-experimental@^7.10.4", "@babel/helper-builder-react-jsx-experimental@^7.11.5":
```
[the remark this answers is not in the document]
Comment (nicolethoen) [reply yarn.lock:212]: Agree. It looks like the versions of things are lower in your updates.

[the remark this answers is not in the document]
Comment (evwilkin) [reply yarn.lock]: Thanks, I'm not quite sure why but I've resolved it now

**Review by redallen — APPROVED** [review]

**Review by jcaianirh — APPROVED** [review]
lgtm

**Review by tlabaj — COMMENTED** [review]
Please do not merge until CSS reviews this one.

**Review by mcoker — APPROVED** [review]
LGTM!

**Review by tlabaj — APPROVED** [review]

**Comment by jcaianirh** [comment]
@mcoker @evwilkin @nicolethoen about this solution. I've updated openshift with the fixed component and core as follows:
`    "@patternfly/patternfly": "4.40",
    "@patternfly/react-core": "4.57.3",
`
With no code adjustments, I get the following wrapping behavior:
Screen Shot 2020-09-28 at 5 27 16 PM

Am I meant to update the DataToolbar to use the prop / value combo: `<Toolbar collapseListedFiltersBreakpoint="xl" ...`
If so I get this behavior:
Screen Shot 2020-09-28 at 5 23 57 PM
Screen Shot 2020-09-28 at 5 23 45 PM

Curious if this is the desired behavior...It seems like the trade off is that we eliminate the chips on a fairly wide view port or keep the wrapped behavior...

**Pull request merged**